lib/person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative 'nameable'
 class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [ ] Once again since we'll need to modify `@rentals` instance variable I'd advise defining both getters and setters.